### DISCUSSION TITLE
Set Wallpaper

### DISCUSSION THREAD

**Discussion by Anon666333** [immich-app/immich#14299] [Feature Request]
I think it would be a great feature to have the ability to set lockscreen/background wallpaper from immich directly. 

Mainly on Android but on IOS as well would be nice 😊

Hopefully I didn't miss this and this is not a repeat discussion. My apologies if it is!

**Comment by bo0tzz** [comment]
It's not quite identical but I'll close this as a duplicate of #10097

**Reply by Anon666333** [reply]
Can we add to that feature request for just setting single wallpapers? Not cycling through multiple images.
Example from Google photos.

![Screenshot_20241123_103108_Photos.jpg](https://github.com/user-attachments/assets/78962821-64b6-43f4-b08b-b7356bc263c8)

**Reply by denysapplications** [reply]
Can this be reopened?
#10097 is clearly not a duplicate of setting and individual picture as a wallpaper.

**Comment by alangrainger** [comment]
## Workaround

I thought I'd add a quick workaround for anyone else looking for this feature. 

1. From Immich, share the photo with the Files app (Google's file app)
2. When the Files app opens, click the most recent image to open it full size.
3. Click the 3-dot menu, then choose **Use as**.

There might be an even more efficient way to do this, but this is the one which worked for me on Pixel and Vivo.
